packages/web/utils/ssr.ts

```diff
@@ -0,0 +1 @@
+export const isBrowser = typeof window !== "undefined";
```

Comment:
Are these primitives really needed as separate files? Why not use them directly inline?

Reply:
I believe it's easier to read and understand thanks to the variable name. What do you think?